### PR TITLE
Transforming this (however roughly) into a Mojo application, with sub…

### LINKED ISSUE #3: Combine the scripts into one, with subcommands
I see you're using [Moo][]. Would you be opposed to using [MooX::Options][] for managing the script CLI? I could also see maybe using [MooX::Cmd][] for a single `brisote` script with subcommands if you wanted to add functionality without cramming too much in users' `bin` folders.

I don't have experience with either library, but I was thinking it might be good to take advantage of somebody else's hard work with the many annoyances of managing a decent CLI. I picked those because I saw "Moo."

[Moo]: https://metacpan.org/pod/Moo
[MooX::Options]: https://metacpan.org/pod/MooX::Options
[MooX::Cmd]: https://metacpan.org/pod/MooX::Cmd

### PR DESCRIPTION
**Pull request by jmacdotorg**
…commands, instead of separate executables. Fixes #3.

### CODE REVIEW

**Comment by jmacdotorg** [comment]
This quite roughly mashes main-branch code into place without any deep attempt to make it more mojo-ey.

We'd also need to plug the hole of it displaying far too much help info by default. Nobody needs to know about 'cpanify', e.g.

File: lib/Whim/Core.pm [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+use warnings;
+use strict;
+use v5.24;
+
+sub startup {
+    my $self = shift;
+
+    # Add another namespace to load commands from
+    push @{ $self->commands->namespaces }, 'Whim::Command';
+}
```
Comment (adherzog) [inline lib/Whim/Core.pm:12]: I think this should probably be removed.

Comment (brianwisti) [reply lib/Whim/Core.pm:12]: Unfortunately, removing this also removes `query` and `send`.

Comment (adherzog) [reply lib/Whim/Core.pm:12]: The namespace is still added in Whim.pm, though. This is in Whim::Core. Unless I'm misunderstanding...

Comment (jmacdotorg) [reply lib/Whim/Core.pm:12]: Adam's right, I accidentally had this same invocation in both Whim.pm and Whim/Core.pm. Removing the entire startup function from Core.pm leaves everything working just fine AFAICT.

**Pull request merged**